### PR TITLE
Renzo/xap-150 Tfm instant dapp library 

### PR DESCRIPTION
**Pull request by renzobarrionuevo**
# 🧙 Description

- Testing the TFM widget using the library and creating a css adapted to work on our website.

Ticket #

## ✅ Checklist

- [ ] Acceptance Criteria described in the ticket are met
- [ ] Texts and strings are in the related i18n file
- [ ] Utilities have unit tests
- [ ] User stories and functionalities have integration or e2e tests
- [ ] If adding a new token or network
  - [ ] Registry package is updated
  - [ ] Asset images are added
- [ ] Related packages.json are upgraded
- [ ] Changelog is updated

### CODE REVIEW

File: packages/instant-dapps/src/dapps/Tfm.tsx [hunk 1]
```diff
@@ -0,0 +1,20 @@
+// Copyright Tharsis Labs Ltd.(Evmos)
+// SPDX-License-Identifier:ENCL-1.0(https://github.com/evmos/apps/blob/main/LICENSE)
+
+"use client";
+
+import Tfm from "@evmosapps/widgets/src/Tfm/tfm";
+import { InstantDappContainer } from "./instant-dapp-container";
+import "@evmosapps/widgets/src/Tfm/tfm.css";
+
+export default function WormholeInstantDapp() {
```
Comment (miliPaoletti) [inline packages/instant-dapps/src/dapps/Tfm.tsx:10]: This should be TfmInstantDapp

File: packages/instant-dapps/src/dapps/Tfm.tsx [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+  return (
+    <div className="widget-container">
```
Comment (miliPaoletti) [inline packages/instant-dapps/src/dapps/Tfm.tsx:12]: Is this div necessary ?

File: packages/widgets/src/Tfm/tfm.css [hunk 3]
```diff
@@ -0,0 +1,2784 @@
+.widget-scope :is(.pointer-events-none) {
```
Comment (miliPaoletti) [inline packages/widgets/src/Tfm/tfm.css:1]: Is .widget-scope still necessary ?

File: packages/widgets/src/Tfm/tfm.tsx [hunk 4]
```diff
@@ -0,0 +1,15 @@
+// Copyright Tharsis Labs Ltd.(Evmos)
+// SPDX-License-Identifier:ENCL-1.0(https://github.com/evmos/apps/blob/main/LICENSE)
+
+import { IBCWidget } from "@0xtfm/widget-ibc";
+import "./tfm.css";
+
+const Tfm = () => {
+  return (
+    <div>
+      <IBCWidget projectId={""} />
```
Comment (miliPaoletti) [inline packages/widgets/src/Tfm/tfm.tsx:10]: Do we have to ask them for a projectId ?

**Review by miliPaoletti — APPROVED** [review]
Looks great! Thanks!!
Could you please also remove the arrow-down /  arrow-up? I think that we are not using those anymore

File: packages/instant-dapps/src/dapps/Tfm.tsx (hunk 2 above)
Comment (renzobarrionuevo) [reply packages/instant-dapps/src/dapps/Tfm.tsx:12]: it is not necessary

**Pull request merged**